src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -63,10 +82,50 @@ public boolean init(Expression<?>[] exprs,
 						ParseResult parseResult,
 						SectionNode sectionNode,
 						List<TriggerItem> triggerItems) {
-		type = ConditionalType.values()[matchedPattern];
-		parseIf = parseResult.mark == 1;
+		type = CONDITIONAL_PATTERNS.getInfo(matchedPattern);
+		ifAny = parseResult.hasTag("any");
+		parseIf = parseResult.hasTag("parse");
+		multiline = parseResult.regexes.size() == 0 && type != ConditionalType.ELSE;
```

Comment:
This can probably be made false for ConditionalType.THEN too right (and avoid checking later on)

Reply:
if it's okay, I think I'd rather have it be `true` for `ConditionalType.THEN`. Originally, I did have it that way but it ended up confusing me a ton since the `then` section "feels" like it's semantically part of the multiline section.